testing/engine_test.go

```diff
@@ -4,14 +4,15 @@
 package testing
 
 import (
+	"fmt"
 	"strings"
 	"testing"
 
 	"github.com/corazawaf/coraza/v3"
 )
 
 func TestRawRequests(t *testing.T) {
-	waf := coraza.NewWAF()
+	waf, _ := coraza.NewWAFWithConfig(coraza.NewWAFConfig())
```

Comment:
I wonder if we could keep both: `NewWAF` and `NewWAFWithConfig`.

Reply:
Ah yeah I was wondering if this could be just `NewWAF`. I thought it could be cool to have a default experience with embedded rules but if we would never have that we don't need two functions (and can add `NewDefaultWAF` or something in the future if that changes)

Reply:
NewDefaultWAF seems more suitable, because it does implement default settings.